lib/OpenQA/Schema/Result/JobGroups.pm

```diff
@@ -156,28 +156,54 @@ sub important_builds {
     return [sort keys %importants];
 }
 
-# list all jobs that are expired
-sub expired_jobs {
-    my ($self) = @_;
+sub _find_expired_jobs {
+    my ($self, $keep_in_days, $keep_important_in_days) = @_;
 
     my @ors;
     my $important_builds = $self->important_builds;
```

Comment:
as this is an quite expensive call, please cache it between results and log cleanup